### DISCUSSION TITLE
PWA click to install update

### DISCUSSION THREAD

**Discussion by danjumacharlesjubril** [quasarframework/quasar#14318] [CLI - PWA mode]
Hi all, I'll like to know how to implement a display of notifications to notify users of the new updates in my quasar PWA, so users can click to update

**Comment by metalsadman** [comment]
https://github.com/quasarframework/quasar/blob/dev/docs/src-pwa/register-service-worker.js

**Reply by danjumacharlesjubril** [reply]
![image](https://user-images.githubusercontent.com/22794718/188328553-796e2f0a-273b-4b54-8170-62fb06e75df2.png)
thanks a lot for your suggestion, I was getting this error when I implemented your answer

**Reply by metalsadman** [reply]
https://quasar.dev/quasar-plugins/notify#installation
